test-docker-compose.yaml

```diff
@@ -63,3 +63,20 @@ services:
       - '.env'
     environment:
       - 'KEEP_ALIVE_PERIOD=0'
+
+  unit_test:
```

Comment:
Container name for e2e tests should also be changed from `test` to `e2e_test` to distinguish it from unit tests.

Reply:
Yes, it can be a follow-up PR.